hail/Makefile

```diff
@@ -125,24 +137,31 @@ fs-jvm-test: upload-remote-test-resources
 .PHONY: jars
 jars: shadowJar shadowTestJar
 
-.PHONY: python-version-info
-python-version-info: $(PYTHON_VERSION_INFO)
-
-python/hail/hail_revision: env/REVISION
-	echo $(REVISION) > $@
-
-python/hail/hail_version: env/SHORT_REVISION env/HAIL_PIP_VERSION
-	echo $(HAIL_VERSION) > $@
+PYTHON_VERSION_INFO := \
+	python/hail/version.py \
+	python/hail/docs/_static/hail_version.js \
+	python/hailtop/version.py
 
-python/hail/hail_pip_version: env/HAIL_PIP_VERSION
-	echo $(HAIL_PIP_VERSION) > $@
+$(PYTHON_VERSION_INFO): env/HAIL_VERSION env/HAIL_PIP_VERSION env/REVISION env/SHORT_REVISION
 
-python/hail/docs/_static/hail_version.js: python/hail/hail_version python/hail/hail_pip_version
-	printf 'hail_version="$(shell cat python/hail/hail_version)";' > $@
-	printf 'hail_pip_version="$(shell cat python/hail/hail_pip_version)"' >> $@
+.PHONY: python-version-info
+python-version-info: $(PYTHON_VERSION_INFO)
 
-python/hailtop/hail_version: python/hail/hail_version
-	cp -f $< $@
+.ONESHELL: $(PYTHON_VERSION_INFO)
+python/hail/version.py python/hailtop/version.py:
```

Comment:
Why duplicate this? The `hail` package depends on `hailtop` and can re-export these constants if necessary or desired.

Reply:
Yeah that's a good point. I guess I just blindly followed what had been done in the past by including a separate hail_version file under `hailtop`. Let me remove this and see if it breaks anything...